server/adaptors/integrations/__data__/repository/nginx/info/README.md

```diff
@@ -0,0 +1,14 @@
+# Nginx Integration
```

Comment:
We should update the nginx configured reference as well

Reply:
done